src/general/cpuInfo.go

```diff
@@ -47,29 +47,51 @@ func NewCPULoad() *CPULoad {
 	return &CPULoad{}
 }
 
+// ReadCPULoad reads /proc/stat and returns the average load
+func ReadCPULoad() ([10]float64, error) {
```

Comment:
Also, maybe `ReadCPULoad` -> `readCPULoad` to make it non-exportable, I'm assuming this is a helper function not being used outside this package.

Reply:
Sure. Will do.